Documentation/flexvolume.md

```diff
@@ -5,34 +5,64 @@ indent: true
 ---
 # Flex Volume Configuration
 Rook uses [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md) to integrate with Kubernetes for performing storage operations. In some operating systems where Kubernetes is deployed, the [default Flexvolume plugin directory](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md#prerequisites) (the directory where flexvolume drivers are installed) is **read-only**.
-
 This is the case for Kubernetes deployments on:
+
+* [Atomic](https://www.projectatomic.io/)
 * [ContainerLinux](https://coreos.com/os/docs/latest/) (previously named CoreOS)
+* [OpenShift](https://www.openshift.com/)
 * [Rancher](http://rancher.com/)
-* [Atomic](https://www.projectatomic.io/)
 
-In these environments, the Kubelet needs to be told to use a different flexvolume plugin directory that is accessible and writeable (`rw`).
-To do this, you will need to first add the `--volume-plugin-dir` flag to the Kubelet and then restart the Kubelet process.
+In these environments, the kubelet needs to be told to use a different flexvolume plugin directory that is accessible and writeable (`rw`).
 These steps need to be carried out on **all nodes** in your cluster.
 
-Please refer to all below sections that are applicable for you, they contain more information on your specific platform and/or Kubernetes version used.
+Please refer to the section that is applicable to your environment, it contains more information on flexvolume on your platform.
 
-```bash
---volume-plugin-dir=/var/lib/kubelet/volumeplugins
-```
-Restart Kubelet in order for this change to take effect.
+## Platform specific flexvolume path
+### For Atomic
+To find out which flexvolume directory path you need to set on the rook-operator in the , please look at the OpenShift docs of your version, [latest OpenShift Flexvolume docs](https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_flex_volume.html#flex-volume-installation) (they also contain the flexvolume path for Atomic hosts).
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
 
-## For Kubernetes >= 1.9.x
-In Kubernetes >= `1.9.x`, you must provide the above set Flexvolume plugin directory when deploying the [rook-operator](/cluster/examples/kubernetes/ceph/operator.yaml) by setting the environment variable `FLEXVOLUME_DIR_PATH`. For example:
+### For ContainerLinux
+Use the [Most common writeable flexvolume path](#most-common-writeable-flexvolume-path) for the next steps.
+
+Continue with [Configuring the flexvolume path](#configuring-the-flexvolume-path) to configure Rook to use the flexvolume path.
+
+### For OpenShift
+See the [For Atomic](#atomic) section.
```

Comment:
in the atomic section, we are actually pointing to the openshift docs.  Therefore, I'd like to move openshift to first in the list and have the openshift docs link actually be in the openshift section.  and then in the Atomic section we should point at the openshift section.